controller-server/src/main/java/com/yahoo/vespa/hosted/controller/Application.java

```diff
@@ -276,4 +276,16 @@ private boolean isPermanent(Environment environment) {
         return true;
     }
     
+    /** Returns true if there is no outstanding change to deploy - i.e deploying is empty or completedly deployed */
```

Comment:
Consider just "change" instead of "outstanding change" as the latter has a different meaning: `Application.hasOutstandingChange()`.